### PR TITLE
Radio Module B update and xdot/sarau260 userland driver code

### PR DESCRIPTION
**Pull request by adkinsjd**
This pull request includes the code necessary for minimal operation of the xdot and the u260 along with two minimal tests of their operation. It also includes an updated board file (the old board revision was moved to the old_revisions folder).

### CODE REVIEW

File: software/apps/libsignpost/sara_u260.c
Comment (bradjc) [inline software/apps/libsignpost/sara_u260.c]: These should be "n" functions, if for no other reason than to set a good example.

File: software/apps/libsignpost/sara_u260.c
Comment (bradjc) [inline software/apps/libsignpost/sara_u260.c]: Delete extra whitespace

File: software/apps/libsignpost/xdot.h
Comment (bradjc) [inline software/apps/libsignpost/xdot.h]: What does this function do?

File: software/apps/libsignpost/xdot.h
Comment (bradjc) [inline software/apps/libsignpost/xdot.h:4]: A one liner comment at the top of the .h files with the name of the chip and a URL would be great.

File: software/apps/radio_module/cellular_test/main.c
Comment (bradjc) [inline software/apps/radio_module/cellular_test/main.c:1]: A couple two line READMEs that explains what should happen in these tests would be great.

File: software/kernel/signpost_drivers/src/sara_u260.rs
Comment (bradjc) [inline software/kernel/signpost_drivers/src/sara_u260.rs]: Don't need this old code.

File: software/kernel/signpost_drivers/src/xdot.rs
Comment (bradjc) [inline software/kernel/signpost_drivers/src/xdot.rs]: here too

**Comment by adkinsjd** [comment]
These commits should address all of Brad's comments.

File: software/apps/libsignpost/multi_console.c
Comment (brghena) [inline software/apps/libsignpost/multi_console.c:44]: Here's the malloc, but where's the free? (same goes for the other three malloc uses)

File: software/kernel/signpost_drivers/src/xdot.rs
Comment (brghena) [inline software/kernel/signpost_drivers/src/xdot.rs:200]: This and the `receive_automatic` call in sara_u260 have different timeouts. You should put a comment to note the difference and explain why it changed from the console driver (in whichever changed).

**Review by brghena — CHANGES_REQUESTED** [review]
Overall looks really good. I've got two changes, but after those I'd say it's good to merge.

**Comment by adkinsjd** [comment]
@brghena Your changes are in the code

**Comment by brghena** [comment]
I'd say it's good to go then when it passes travis.

**Pull request merged**
